resources/lang/zho.ini

```diff
@@ -1,5 +1,5 @@
 ; Updated time : 26th 07 2017
-language.name = "Traditional Chinese"
```

Comment:
wasn't this changed to remove confusion?

Reply:
I revert the change, then changed zho.ini to "Traditional Chinese".... right now the "zho.ini" should be Traditional Chinese and the "chi.ini" is "Simplified Chinese". I just realized I did the opposite of what the post said...